.changeset/tripwire-notice-light-mode.md

```
@@ -0,0 +1,5 @@
+---
+'@internal/playground': patch
+---
+
+Fixed the "Content Blocked" tripwire notice in Studio so it uses the shared Notice component, making its colors readable in light mode.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)